package.json

```diff
@@ -38,6 +38,7 @@
     "@octokit/rest": "^19.0.3",
     "@types/semver": "^7.3.10",
     "moment": "^2.29.4",
+    "proxy-agent": "^5.0.0",
```

Comment:
I understand that the used dependency is suggested by the `octokit` docs - but the latest version has some unpatched issues open - https://github.com/TooTallNate/node-proxy-agent/issues/74

Need to see if there are alternatives.